src/content/release/breaking-changes/uiscenedelegate.md

```diff
@@ -625,6 +510,129 @@ sceneLifeCycleDelegate.unregisterSceneLifeCycle(with: flutterEngine)
 [self.sceneLifeCycleDelegate unregisterSceneLifeCycleWithFlutterEngine:self.flutterEngine];
 ```
 
+## Migration guide for Flutter plugins
+
+Not all plugins use lifecycle events. If your plugin does, though, you will
+need to migrate to UIKit's scene-based lifecycle.
+
+1. Update Flutter SDK version in your pubspec.yaml
+
+:::warning
+The below Flutter APIs are available in the 3.38.0-0.1.pre beta, but are not
+yet available on stable. You may consider publishing a
+[prerelease version](https://dart.dev/tools/pub/publishing#publishing-prereleases)
+of your plugin to migrate early.
+:::
```

Comment:
@stuartmorgan-g What do you think of this?

Reply:
Are you sure previews work for the Flutter SDK? The docs make it sound specific to the Dart SDK version

Reply:
I'm not 100% sure that it trigger the pub.dev preview if you only set the Flutter SDK, but you could just set both: the Flutter SDK to the version you need, and the Dart SDK to the version of Dart that shipped with that version of Flutter.

Reply:
Sounds good, updated.